src/create.cpp

```diff
@@ -1,14 +1,14 @@
 //
 // Created by ENDERZOMBI102 on 15/10/2023.
 //
+#include <crc32.h>
 #include <filesystem>
 #include <fmt/chrono.h>
 #include <fmt/format.h>
 #include <fstream>
-#include <hash-library/crc32.h>
```

Comment:
This is a compile error on linux, as system includes take precedence, `/usr/include/crc32` is included instead of the `hash-library` one

Reply:
That sucks lol